### PR TITLE
Erika A Porath - Project Todos

### PR DESCRIPTION
**Pull request by TheErry**
Link to my deployed project: 
https://errys-todos.netlify.app/

My reflections on how this week's project turned out: 
This project was quite straight-forward and I didn't have many issues with it. I actually finished the blue goals quite early, but unfortunately I have other things I need to deal with this week and still doesn't have time to try the advanced goals.

Things I'd like to have clarified or explained in more detail:
I names some of the components .jsx to specify that they're React components, and because I think it looks pretty, but I forgot to do it to some or they were already created as .js. I couldn't rename them later because I got errors that the files weren't found then, so I left it as it is by now even though I know it isn't optimal.

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,82 @@
-import React from 'react'
+import React from "react"
+import { Provider } from "react-redux"
+import { combineReducers, configureStore } from "@reduxjs/toolkit"
+import styled, { createGlobalStyle } from "styled-components"
+
+import GlobalFonts from "./fonts/fonts"  //Comes from createGlobalstyle in fonts
```
Comment (Marielle8) [inline code/src/App.js:6]: Good job with the import of fonts!

File: code/src/components/AddTask.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  
+  const TextInput = styled.input`
+  border: none;
+  background: #FDF4C9;
+  padding: 10px;
+  color: #2C2E73;
+  font-family: "WrittenLies";
+  font-size: 20px;
+  width: 80%;
+`
```
Comment (Marielle8) [inline code/src/components/AddTask.jsx:33]: Indendation...🤓

File: code/src/components/AddTask.jsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  }
+
+  return (
+    <StyledForm onSubmit={onFormSubmit}>
+      <TextInput
+        type="text"
+        value={value}
+        onChange={e => setValue(e.target.value)}
+        aria-label="text input"
+        required
```
Comment (Marielle8) [inline code/src/components/AddTask.jsx:70]: Good thinking to do the textinput required and to add aria-label!

File: code/src/components/Header.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+const StyledHeader = styled.div`
+  display: flex;
+  justify-content: center;
+  color: #2C2E73;
+  width: 100%;
+  text-align: center;
+  font-family: "WrittenLies";
+  `
+
+  const Title = styled.h1`
```
Comment (Marielle8) [inline code/src/components/Header.js:14]: Indendation...🤓

File: code/src/components/TaskSummary.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+const TaskSummary = () => {
+  const allTasks = useSelector(store => store.todos.items)
+  const uncompletedTasks = allTasks.filter(item => item.isComplete === false).length
+
+  return (
+    <Summary>
+      <p>{uncompletedTasks} of {allTasks.length} task left to do!</p>
+    </Summary>
+  )
```
Comment (Marielle8) [inline code/src/components/TaskSummary.js:19]: Very nice to show both the total tasks and the uncompleted tasks!

File: code/src/components/TodoList.jsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+  
+  const StyledParagraph = styled.p`
+  margin: 0;
+  padding: 0px 10px 0px 10px;
+`
+
+const Checkbox = styled.input`
+  cursor: pointer;
+  width: 20px;
+  height: 20px;
```
Comment (Marielle8) [inline code/src/components/TodoList.jsx:39]: When you write a to do that is longer than one line the checkbox gets a smaller  than what it is from the beginning. Maybe that is something to look at if you feel like you want to continue with the project in the future.

File: code/src/fonts/fonts.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import { createGlobalStyle } from "styled-components"  //Imported createGlobalStyle to be able to use the fonts everywhere
```
Comment (Marielle8) [inline code/src/fonts/fonts.js:1]: Good with the comment 👍

File: code/src/components/AddTask.jsx [hunk 8]
```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react"
+import { useDispatch } from "react-redux"
+import uniqid from "uniqid"
+import styled from "styled-components"
+
+import todos from "../reducers/todos"
+
+//Styled components
+const StyledForm = styled.form`
```
Comment (Marielle8) [inline code/src/components/AddTask.jsx:9]: Good job with the styled components😍

**Review by Marielle8 — APPROVED** [review]
Very nice project Erika! The styling is awesome, I love the idea with a hand-written to do list! Nice that it is responsive as well! What I can see you have fulfilled all the blue requirments! Great job🦄! Your code is clean and easy to follow🙌

**Review by KarolCislowski — APPROVED** [review]
Great project 🥇

**Pull request closed without merging**